### DISCUSSION TITLE
Navigation to a type on Right Ctrl + Click into the popup showing function definition

### DISCUSSION THREAD

**Discussion by Alex2357** [ionide/ionide-vscode-fsharp#1189] [Ideas]
If I mouse over the function usage or a declaration it shows a popup with function definition, if I press Right Ctrl and click into the one of the types appearing in that popup nothing happens.

I'd like to be navigated to the type definition I clicked

[a comment by an automated account is not shown]

**Comment by MangelMaxime** [comment]
Hello, @Alex2357 we can't support this navigation yet in Ionide because we are using LSP and by default, it doesn't allow to run a command from Tooltip.

I created an issue related to this limitation in https://github.com/microsoft/vscode-languageserver-node/issues/503 and one of the maintainers mentioned he would accept a PR to add the feature. I should soon send a PR to their project.

**Comment by MangelMaxime** [comment]
For information, the issue has just been resolved in the repository. 

So we need to wait for it to land into VSCode and then we will be able to use the new feature.
